BuildTools/BuildTools.proj

```diff
@@ -14,6 +14,6 @@
     <PackageReference Include="Microsoft.Windows.SDK.Win32Metadata" Version="$(LastWin32MetadataReleaseVersion)" GeneratePathProperty="True">
       <PrivateAssets>all</PrivateAssets>
     </PackageReference>
-    <PackageReference Include="Microsoft.Direct3D.D3D12" Version="1.616.0" GeneratePathProperty="true"/>
+    <PackageReference Include="Microsoft.Direct3D.D3D12" Version="1.618.1" GeneratePathProperty="true"/>
```

Comment:
Note that I don't have access to update the NuGet feed, @mikebattista can you do this so that users don't run into https://github.com/microsoft/win32metadata/issues/2105 again?